forge/ee/db/models/TeamBrokerUser.js

```diff
@@ -0,0 +1,79 @@
+const { DataTypes } = require('sequelize')
+
+const { hash, buildPaginationSearchClause } = require('../../../db/utils')
+
+module.exports = {
+    name: 'TeamBrokerUser',
```

Comment:
Should this be `TeamBrokerClient`?

Reply:
It was written early on before we started talking in terms of clients, but I've changed it.